controllers/cryptoController.js

```diff
@@ -0,0 +1,90 @@
+const usersdata = require('../models/crypto')
+const calculation = require('../middlewares/cryptoTransactionClac')
+/**
+ * Send Money from one User to Other
+ * @param {Object} req - Express request object
+ * @param {Object} res - Express response object
+ */
+const send = async (req, res) => {
+  try {
+    const request = req.body
+    const [userTo, userFrom, currency, amount] = [request.to, request.from, request.currency, request.amount]
+    const message1 = `${amount} ${currency} Coins Transfered From ${userFrom} to ${userTo}`
+    const verifyUserTO = await usersdata.fetchUser(userTo)
+    const verifyUserFrom = await usersdata.fetchUser(userFrom)
+    const resultObj = calculation.cryptoCalc(verifyUserTO, verifyUserFrom, currency, amount)
+    return res.json({
+      message: message1,
+      to: resultObj.verifyUserTO,
+      from: resultObj.verifyUserFrom
+    })
+  } catch (error) {
+    logger.error(`Error while processing pull requests: ${error}`)
+    return res.boom.badImplementation('An internal server error occurred')
+  }
+}
+
+/**
+ * Receive Money from one User to Other
+ * @param {Object} req - Express request object
+ * @param {Object} res - Express response object
+ */
+const request = async (req, res) => {
+  try {
+    const request = req.body
+    const [userTo, userFrom, currency, amount] = [request.to, request.from, request.currency, request.amount]
+    const notificationMessage = `${amount} ${currency} Coins Requested By ${userFrom}`
+    const message1 = `${amount} ${currency} Coins Requested By ${userFrom} From ${userTo}`
+    const verifyUserTO = await usersdata.fetchUser(userTo)
+    const verifyUserFrom = await usersdata.fetchUser(userFrom)
+    if (verifyUserFrom && verifyUserTO) {
+      const notificationArray = verifyUserTO.user.notification
+      notificationArray.push(notificationMessage)
```

Comment:
Does this push creates a new  notification in db  also.

Reply:
Yes I have attached screenshot of collection. we have notification array.